src/biogeochem/DryDepVelocity.F90

```diff
@@ -654,6 +664,60 @@ subroutine depvel_compute( bounds, &
                rc = 1._r8/((1._r8/rsmx(ispec))+(1._r8/rlux(ispec)) + &
                     (1._r8/(rdc+rclx(ispec)))+(1._r8/(rac(index_season,wesveg)+rgsx(ispec))))
                rc = max( 10._r8, rc)
+
```

Comment:
I can try to do that, I'm not really familiar with the CTSM coding convention though so not sure how to create this 2-D variable with season and PFT.

Reply:
Oh, I didn't realize this was a time varying variable.  Since this is something we're already calculating elsewhere in the model, should we just use the ratio of SAI and LAI for consistency?  I realize this may change answers scientifically that you'd want to evaluate?  The easier option would be to keep this _as is_ for CLM6/CESM3?

Reply:
leave it as it is for now

Reply:
The model has another sai = stem area index. Better to call this one stai to prevent confusion?